gtsfm/data_association/point3d_initializer.py

```diff
@@ -291,17 +303,19 @@ def extract_measurements(self, track: SfmTrack2d) -> Tuple[CameraSetCal3Bundler,
         track_cameras = CameraSetCal3Bundler()
         track_measurements = Point2Vector()  # vector of 2d points
 
+        n_meas = 0
```

Comment:
i prefer `num_valid_measurements = 0` since `n_meas` seems like n measured or n measurements or meas just isnt super clear to me